test/test-debuglet.js

```diff
@@ -309,10 +308,14 @@ describe('Debuglet', function() {
 
 
     it('should pass source context to api if present', function(done) {
-      process.chdir(path.join(__dirname, 'fixtures'));
-
       var debug = require('..')(
           {projectId: 'fake-project', credentials: fakeCredentials});
+      var old = Debuglet.prototype.getSourceContext_;
+      Debuglet.prototype.getSourceContext_ = function(cb) {
+        setImmediate(function () {
+          cb(null, {a: 5});
```

Comment:
Do we want to delete the source context in fixtures as well if it is no longer used?